about.html

```diff
@@ -0,0 +1,202 @@
+          <a href="#" class="text-white hover:text-light-gray">
+            <span class="bi bi-facebook"></span>
+          </a>
+        </div>
+      </div>
+      <nav class="navbar navbar-expand-md navbar-light bg-white shadow py-2" id="top-navbar">
+        <div class="container">
+          <button
+            class="navbar-toggler border-0"
+            type="button"
+            id="navbar-btn"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarNav"
+            aria-controls="navbarNav"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <a class="navbar-brand d-none d-sm-block" href="./">
+            <img src="./images/logo.png" alt="" height="51" />
+          </a>
+          <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav ms-auto fw-bold">
+              <li class="nav-item me-5">
+                <a class="nav-link" aria-current="page" href="./about.html">About</a>
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link" href="#">Program</a>
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link" href="#">Join</a>
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link" href="#">Sponser</a>
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link">News</a>
+              </li>
+              <li class="nav-item">
+                <button class="nav-link btn btn-outline-primary rounded-0 text-orange hover:text-light-gray px-3">
+                  Join Now
+                </button>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="hero">
+        <div class="container py-5">
+          <div class="hero-content mx-auto">
+            <div class="py-2 py-md-5"></div>
+            <h3 class="text-orange text-center h3 d-sm-none">"Hello, Working World!"</h3>
+            <h1 class="text-orange font-cg-regular hero-title text-center">COWORKING SPACE YOUR DREAMS OFFICE</h1>
+            <div class="small-border mx-auto mt-3 mb-5"></div>
+            <div class="hero-text bg-white py-4 text-gray font-cg-light fs-7 text-center border-light">
+              Lorem ipsum dolor sit amet consectetur, adipisicing elit. Accusantium, suscipit molestiae? Velit corrupti
+              numquam libero impedit. Debitis, in? Ipsam soluta error adipisci corrupti saepe est vero id ad
+              necessitatibus voluptates.
+              <p>
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit. Optio hic veniam adipisci, tenetur eaque vitae
+                sed quas nesciunt aspernatur dolore nobis repellat provident, in, ut repellendus similique harum vel
+                non.
```

Comment:
- [ ] Please make sure that the text matches the one in the Behance design 

| Your design | Behance |
| ------------- | --------- |
| ![image](https://user-images.githubusercontent.com/74744787/163285070-eba63081-1a5e-49aa-acdd-29903aa2a690.png) | ![image](https://user-images.githubusercontent.com/74744787/163285133-2fd52b32-478a-475a-980b-7083e0882bc2.png) |
| Please make sure that you are using the same font color ,the same text,the same font family (Cocogoose, Lato)  |